Composer/packages/ui-plugins/virtual-assistant-creation/src/components/provisionSummaryPage.tsx

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { Fragment } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { DetailsList, DetailsListLayoutMode, IColumn, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+import { RouterPaths } from '../shared/constants';
+
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+
+// Content Data
+const provisioningDetailItems = () => {
```

Comment:
Does this need to be function or can it be a const value?

Reply:
changed to const

Reply:
This needs to be wrapped in a function in order for `formatMessage` to pick up the current locale.

Reply:
Changed to function :D